pkg/cmd/grafana-cli/commands/mergefs_test.go

```diff
@@ -0,0 +1,44 @@
+package commands
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"testing/fstest"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMergeFS(t *testing.T) {
+	tempDir := os.DirFS(filepath.Join("testdata", "mergefs"))
+	a := fstest.MapFS{
+		"a/":          &fstest.MapFile{Data: []byte("a")},
+		"a/z/":        &fstest.MapFile{Data: []byte("a/z")},
+		"a/z/foo.cue": &fstest.MapFile{Data: []byte("foo")},
+	}
+
+	filesystem := Merge(a, tempDir)
+	require.NoError(t, fs.WalkDir(filesystem, ".", func(path string, d fs.DirEntry, err error) error {
```

Comment:
per above comment, probably shouldn't rely on WalkDir for testing. or at least, not all testing. The question is all about what properties we're trying to demonstrate the system has, and finding the simplest way to do so. (Do i sounds like a broken record yet? 😜)